modules/configuring-vsphere-host-groups.adoc

```diff
@@ -0,0 +1,128 @@
+// Module included in the following assemblies:
+//
+//* /installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned-customizations.adoc
+//* /installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned-network-customizations.adoc
+//* /installing/installing_vsphere/ipi/installing-restricted-networks-installer-provisioned-vsphere.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-vsphere-host-groups_{context}"]
+= Configuring host groups for a VMware vCenter
+
+:FeatureName: Openshift zones support for vSphere host groups 
+include::snippets/technology-preview.adoc[]
+
+You can modify the default installation configuration file to deploy an {product-title} cluster on a vSphere stretched cluster, with ESXi hosts and storage distributed across multiple physical datacenters.
+
+The default `install-config.yaml` file configuration from the previous release of {product-title} is deprecated. Though you can still use it, the openshift-installer will display a warning message that indicates the use of deprecated fields in the configuration file.
```

Comment:
I don't see any uses of _openshift-installer_ in the docs after a quick Atom search. 
```suggestion
The default `install-config.yaml` file configuration from the previous release of {product-title} is deprecated. Though you can still use it, the {product-title} installer displays a warning message that indicates the use of deprecated fields in the configuration file.
```